UsbAccess/doc/UsbAccessPlugin.md

```diff
@@ -368,14 +401,61 @@ This method takes no parameters.
 ```json
 {
     "jsonrpc": "2.0",
-    "id": 1234567890,
+    "id": 42,
     "result": {
         "success": true,
         "error": "invalid filename"
     }
 }
 ```
 
+<a name="method.ArchiveLogs"></a>
+## *ArchiveLogs [<sup>method</sup>](#head.Methods)*
+
+(Version 2) Creates a log directory and archives the logs files of the device.
```

Comment:
Pls change description to indicate that logs will be uploaded to USB. 
Something like "Compresses and uploads device logs into attached USB drive"

Can we also give details on which location the logs will go into.

Reply:
Updated the changes as per your review comments.